common/adminBlock/EntryAddFactoidAddress.go

```diff
@@ -11,16 +11,18 @@ import (
 	"github.com/FactomProject/factomd/common/primitives"
 )
 
-// AddFactoidAddress Entry -------------------------
+// AddFactoidAddress is an admin block entry containing a server identity and a factoid address where coinbase payouts will be sent for that
+// server
 type AddFactoidAddress struct {
-	AdminIDType     uint32 `json:"adminidtype"`
-	IdentityChainID interfaces.IHash
-	FactoidAddress  interfaces.IAddress
+	AdminIDType     uint32              `json:"adminidtype"` // the type of action in this admin block entry: uint32(TYPE_ADD_FACTOID_ADDRESS)
+	IdentityChainID interfaces.IHash    // the server identity
+	FactoidAddress  interfaces.IAddress // the factoid address for this server
```

Comment:
Should add json tags to all files, or none of them.  To be consistent.

Reply:
I was deliberately not adding json tags that were missing, because i wasn't sure if adding them would screw up any backwards compatibility, but I can go back and add them when i see they are missing.

Reply:
I added the tags, you can relook when i push the changes